File: flax_search_service/docs/tutorial/python/t2_show_schema.py
import flax.searchclient

# open a connection to Flax Server
conn = flax.searchclient.Client('http://localhost:8080')

# get a reference to the schema
schema = conn.schema('books')

# display the field definitions
for field in schema.get()['fields'].iteritems():
    print field

File: flax_search_service/docs/tutorial/php/t2_show_schema.php
<?php
include_once "flax/flaxclient.php";

# open a connection to Flax Server
$conn = new FlaxSearchService('http://localhost:8080');

# get a reference to the database
$db = $conn->getDatabase('books');

# display the field definitions
foreach ($db->getFieldNames() as $fname)
{
    echo $fname .": ". $db->getField($fname) ."\n";
}

?>